openraft/src/vote/raft_term/mod.rs

```diff
@@ -0,0 +1,21 @@
+mod raft_term_impls;
+
+use std::fmt::Debug;
+use std::fmt::Display;
+
+use crate::base::OptionalFeatures;
+
+/// Type representing a Raft term number.
+///
+/// A term is a logical clock in Raft that is used to detect obsolete information,
+/// such as old leaders. It must be totally ordered and monotonically increasing.
+///
+/// Common implementations are provided for standard integer types like `u64`, `i64` etc.
+pub trait RaftTerm
```

Comment:
Module `vote` is private, so this trait is private as well, this is the reason why the lint CI failed